packages/@react-stately/data/docs/useAsyncList.mdx

```diff
@@ -129,6 +129,10 @@ This example shows how to implement client side sorting by passing a `sort` func
 items array.
 
 ```tsx
+import { useCollator } from '@react-aria/i18n';
+
+let collator = useCollator();
+
 let list = useAsyncList({
```

Comment:
Small nit pick for consistency, but I think we can get away with not putting this import here since this is a tsx snippet, not a live example
```suggestion
let collator = useCollator();
let list = useAsyncList({
```